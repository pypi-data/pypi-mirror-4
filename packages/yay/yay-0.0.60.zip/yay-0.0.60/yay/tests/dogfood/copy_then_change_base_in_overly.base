foo:
  bar:
    - baz
    - qux
zop.copy: ${foo.bar}

