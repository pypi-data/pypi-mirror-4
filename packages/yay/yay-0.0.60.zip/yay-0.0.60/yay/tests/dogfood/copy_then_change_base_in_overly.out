foo:
  bar:
    - x

zop:
  - x

